stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/messages/view/MessageListView.kt

```diff
@@ -225,7 +228,8 @@ public class MessageListView : ConstraintLayout {
                         message,
                         messageOptionsConfiguration.copy(
                             threadEnabled = !adapter.isThread && !message.isInThread(),
-                        )
+                        ),
+                        messageListViewStyle.itemStyle
```

Comment:
```suggestion
                        messageListViewStyle.itemStyle,
```

Reply:
Haha good catch. You've got some eagle eyes!